src/components/Pagination/Pagination.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../SFE.scss';
+@import "~bootstrap/scss/utilities/_screenreaders.scss";
+
+.pagination {
+    justify-content: center;
+    align-items: center;
+}
+
+.page-item {
+    min-width: 50px;
+    text-align: center;
+}
+
+.disabled {
```

Comment:
We're trying to leverage the bootstrap classnames for pagination for the pagination component. The [scss](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_pagination.scss#L45) expects that a disabled pagination "button" has a page-item parent with a page-link child that has the disabled classname on the page-link. However, the pagination package we're using puts the disabled class name on the page-item parent instead of the page-link class. Therefore, we have to create this disabled class that we can use as a classname on the [page-link](https://github.com/edx/studio-frontend/pull/54/files#diff-19113b55aba41492c4d2adb0959b1478R77).